adapters/repos/db/roaringset/helpers_test.go

```diff
@@ -34,12 +34,15 @@ func TestBitmap_Condense(t *testing.T) {
 		condensed := Condense(bm)
 		condensedLen := len(condensed.ToBuffer())
 
-		assert.Greater(t, bmLen, condensedLen)
+		// As of sroar 0.0.5 "And" merge is optimized not to expand
```

Comment:
I think it should be: `0.0.6`

Reply:
well, changes were added to 0.0.5, 0.0.6 included just fixes